package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "bedrock-permission-http",
+  "version": "1.0.1-dev",
```

Comment:
This version should be like "0.0.1-dev" if we haven't released yet. Don't put anything over 1.0.0.